apps/web/src/pages/SearchRequests/SearchPage/components/SearchForm.tsx

```diff
@@ -107,6 +112,53 @@ export const SearchForm = ({
     setValue("count", candidateCount);
   };
 
+  const hireAnApprenticeEmailUri = buildMailToUri(
+    "edsc.patipa.jumelage.emplois-itapip.job.matching.esdc@hrsdc-rhdcc.gc.ca",
+    intl.formatMessage({
+      defaultMessage: "I'm interested in offering an apprenticeship",
+      id: "HqtjhD",
+      description: "Subject line of a manager's email for apprenticeship",
+    }),
+    [
+      intl.formatMessage({
+        defaultMessage:
+          "To best support you in your journey to hire an IT Apprentice, please let us know if you",
+        id: "ZKss5S",
+        description: "Paragraph 1 of a manager's email for apprenticeship",
+      }),
+      intl.formatMessage({
+        defaultMessage:
+          "1. are interested in hiring an Apprentice and would like to learn more about the IT Apprenticeship Program for Indigenous Peoples",
```

Comment:
1. are interested in hiring an apprentice and would like to learn more about the IT Apprenticeship Program for Indigenous Peoples
- If not out of scope, it should ideally have a lower case a for the first occurrence of apprentice